frontend/src/components/FullCommentList/FullCommentList.tsx

```diff
@@ -24,41 +24,43 @@ const FullCommentList = ({
 }: Props) => {
 	const virtuoso = useRef<VirtuosoHandle>(null)
 	const { loading: isLoadingSlack } = useSlackBot()
-
 	const loading = isLoadingComments || isLoadingSlack
+
 	return (
 		<Box display="flex" flexDirection="column" height="full">
 			{loading && <LoadingBox />}
-			{!loading && comments.length === 0 ? (
-				<div className={styles.noCommentsContainer}>
-					<div className={styles.noCommentsTextContainer}>
-						<h2>There are no comments yet</h2>
-						<p>{noCommentsMessage}</p>
-					</div>
-					<ConnectHighlightWithSlackButton />
-				</div>
-			) : (
+			{!loading && (
```

Comment:
This change fixes the loading UI so it is centered vertically in the panel. We were showing the list component while in the loading state causing it to be pushed up from the center.